master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -2,37 +2,31 @@
 title: Accessing Calico policy with Calico as a network plugin
 ---
 
-
 ## Background
 
-With Calico, a Docker network represents a logical set of rules that define the 
-allowed traffic in and out of containers assigned to that network.  The rules
-are encapsulated in a Calico "profile".
-
-When creating a Docker network using the Calico network driver, the Calico 
-driver creates a profile object for that network.  The default policy applied 
-by Calico when a new network is created allows communication between all 
-containers connected to that network, and no communication from other networks.
+With Calico as a Docker network plugin, network policy may be applied in two ways:
 
-Using the standard `calicoctl create` to create a `profile` resource, it is possible to 
-manage the feature-rich policy associated with a network.
+-   Policy is defined at the network scope and applied directly to each endpoint
+    in the network.
+-   Labels (key/value pairs) are defined at the network scope and applied to each 
+    endpoint in the network;  a global set of policy is defined that uses selectors
+    to determine which subset of the policy is applied to those endpoints based on
+    the labels.
 
-> Note that if you want to access the feature rich Calico policy, you must use
-> both the Calico Network _and_ Calico IPAM drivers together.  Using the Calico
-> IPAM driver ensures _all_ traffic from the container is routed via the host
-> vRouter and is subject to Calico policy.
-
-> The profile that is created by the Calico network driver is given the same 
-> name as the Docker network name.
+A Docker network created using Calico has an identically name Calico profile 
+associated with it which the Calico plugin applies to each endpoint in the network.
+If the profile does not exist the Calico plugin will automatically create it with 
```

Comment:
Leave this out, I think it's an answer to a question most people won't be asking when they read this